nestkernel/per_thread_bool_indicator.h

```diff
@@ -106,6 +108,34 @@ class PerThreadBoolIndicator
 
   BoolIndicatorUInt64& operator[]( const size_t tid );
 
+  void
+  set_true( const size_t tid )
+  {
+    if ( per_thread_status_[ tid ].is_false() )
+    {
+      are_true_++;
+      per_thread_status_[ tid ].set_true();
+    }
+  }
+  void
+  set_false( const size_t tid )
+  {
+    if ( per_thread_status_[ tid ].is_true() )
+    {
+      are_true_--;
+      per_thread_status_[ tid ].set_false();
+    }
+  }
+  void
+  logical_and( const size_t tid, const bool status )
+  {
+    if ( per_thread_status_[ tid ].is_true() && !status )
```

Comment:
It is NEST convention to always write out boolean operators:
```suggestion
    if ( per_thread_status_[ tid ].is_true() and not status )
```